tests/functional/translate/cram/_setup.sh

```diff
@@ -0,0 +1,2 @@
+pushd "$TESTDIR/../../" > /dev/null
+export AUGUR="${AUGUR:-../../bin/augur}"
```

Comment:
_(non-blocking)_

It would be nice to have just one of these files (the same as [tests/functional/filter/cram/_setup.sh](https://github.com/nextstrain/augur/blob/67027a6ae6a7a324c8eb646593cc5c0a89650ccf/tests/functional/filter/cram/_setup.sh)), but this is fine for now.